components/OHPC_macros

```diff
@@ -115,9 +116,32 @@ Requires:      openmpi3-%{compiler_family}%{PROJ_DELIM}
 %endif
 %endif
 
+# Python dependencies and macros
+%if 0%{?ohpc_python_dependent} == 1
+%if "%{python_family}" == "python3"
+%global __python %__python3
```

Comment:
Where is %__python3 defined?

Reply:
it's in EPEL's python3-rpm-macros, a requirement of python34-devel